rclcpp/include/rclcpp/node_interfaces/node_graph.hpp

```diff
@@ -61,6 +61,11 @@ class NodeGraph : public NodeGraphInterface
 
   RCLCPP_PUBLIC
   virtual
+  std::vector<std::string>
```

Comment:
Not sure about this. I guess in terms of performance, a vector or list don't differ that much and the underlying C-type is a `char **`. Depending on the application, we may actually want to have indexed access (such as exposing all topics attached to a individual rosnode).